recipes-omnect/omnect-device-service/omnect-device-service_0.17.1.bb

```diff
@@ -6,9 +6,9 @@ inherit cargo
 # DEFAULT_PREFERENCE = "-1"
 
 # how to get omnect-device-service could be as easy as but default to a git checkout:
-# SRC_URI += "crate://crates.io/omnect-device-service/0.17.1"
-SRC_URI += "git://github.com/omnect/omnect-device-service.git;protocol=https;nobranch=1;branch=main"
-SRCREV = "b7d3a4184ea45ae4f510da345c5fe0a42f73a113"
+# SRC_URI += "crate://crates.io/omnect-device-service/0.14.16"
+SRC_URI += "git://github.com/HarryWaschkeit/omnect-device-service.git;protocol=https;nobranch=1;branch=hwt-healthcheck"
```

Comment:
wrong source URI

Reply:
is there a reason to not create the omnect-device-service PR?

Reply:
corrected.